docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -0,0 +1,133 @@
+# Monitoring
+
+Monitoring is the ability of gathering data and insights of the execution of an application, then allowing users to inspect the gathered data and determine potential actions to take depending on the data collected.
+
+In version vX.Y.Z of the AWS JDBC Driver, we have introduced a Telemetry feature to the wrapper that enables users to collect and visualize data of the execution of the AWS JDBC Driver, at a global level and at plugin level, meaning that one can now inspect specific performance data for each plugin in the plugin chain.
```

Comment:
```suggestion
In version X.Y.Z of the AWS JDBC Driver, we have introduced a Telemetry feature to the wrapper enabling users to collect and visualize data of the AWS JDBC Driver execution, at a global level and at plugin level, meaning that one can now inspect specific performance data for each plugin in the [plugin chain](../development-guide/Pipelines.md).
```

Reply:
We never really explained what a plugin chain is in our documentation, not sure if it makes sense to mention it.